packages/axes/src/eventInput/MouseEventInput.ts

```diff
@@ -45,8 +45,8 @@ export class MouseEventInput extends EventInput {
     return;
   }
 
-  public getTouches(): number {
-    return 0;
+  public getTouches(event: InputEventType): number {
```

Comment:
Can you think of dragging with the right mouse button and dragging with the wheel button?

Reply:
You pointed out a good point I was missing.
To avoid other unexpected exceptions, this statement should check if `event.which` is included in the list of inputButton.